src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -99,63 +99,87 @@ public override ICommandLineParserResult ParseArgs(string[] args)
             return Parser.Parse(args);
         }
 
+        public static IWebHost BuildWebHost(string url, IConfiguration configuration) =>
+            Microsoft.AspNetCore.WebHost.CreateDefaultBuilder(new string[0])
+            .UseConfiguration(configuration)
```

Comment:
This shows how configuration is being passed in now. This will flow all the way down to the WebJobs SDK so we shouldn't have to deal with environment variable or config manimpulation. It might still be needed to make sure **user** code that was relying on that still works and have a way to get to that data, though.